plugins/inspect/services_inspector.rb

```diff
@@ -85,4 +97,29 @@ def inspect_sysvinit_services(system)
 
     ServiceList.new(services.sort_by(&:name))
   end
+
+  def inspect_sysvinit_redhat_services(system)
+    system.check_requirement("chkconfig", "--version")
+    system.check_requirement("runlevel")
+
+    output = system.run_command(
+      "chkconfig", "--list",
+      :stdout => :capture
+    )
+
+    previouslevel,runlevel = system.run_command(
+    "runlevel",
+      :stdout => :capture
+    ).split(" ")
+
+    @summary = "Found #{output.lines.size} services."
+
+    services = output.lines.map do |line|
+    state=[]
+    name, state[0], state[1], state[2], state[3], state[4], state[5], state[6] = line.split(/\s+/)
+    Service.new(name: name, state: state[runlevel.to_i].split(":")[1])
+  end
```

Comment:
`end` at 121, 2 is not aligned with `services = output.lines.map do |line|` at 117, 4